DFC.ServiceTaxonomy.Editor/Recipes/36. Taxonomies_39f603d6-746a-4324-b59a-31924e5d44fb.recipe.json

```diff
@@ -99,6 +99,36 @@
                                             "TermPart": {}
                                         }
                                     },
+                                    {
```

Comment:
This is a generated recipe and will get replaced once the `GetJobProfiles` utility is next run and the recipes checked in.

The `GetJobProfiles` utility should be updated to output these three new/updated recipes.